websites/S/Stadia/9anime/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "ririxi",
+    "id": "192490421308489731"
+  },
+  "service": "9anime",
+  "description": {
+    "en": "9anime is a streaming website where you can watch anime plus dubbed audio in English.",
+    "zh_CN": "9anime是一个动漫视频网站，您可以在其中以英语来观看各种动漫。",
+    "zh_TW": "9anime是一個動漫視頻網站，您可以用英語配音來觀看各種動漫。",
+    "ga_IE": "Is suíomh Gréasáin sruthú é 9anime inar féidir leat féachaint ar anime móide fuaim dubáilte i mBéarla.",
+    "nl": "9anime is een streamingwebsite waar je anime plus nagesynchroniseerde audio in het Engels kunt bekijken."
+  },
+  "url": "9anime.to",
+  "version": "1.1.10",
+  "logo": "https://i.imgur.com/jPl7EfZ.png",
+  "thumbnail": "https://i.imgur.com/RIGfshD.png",
+  "color": "#694ba1",
+  "tags": [
+    "embedded-links",
+    "9anime",
+    "streaming"
+  ],
+  "category": "anime",
+  "regExp": "([a-z0-9-]+[.])*9anime([.][a-z]+)+[/]",
+  "iframe": true
+}
```

Comment:
what did you did? XD remove this folder. Why 9anime is in Stadia folder

Reply:
LMAO OOPS